sources/packages/forms/src/form-definitions/sfaa2025-26.json

```diff
@@ -10905,6 +10905,49 @@
               "lazyLoad": false,
               "tags": [],
               "hideLabel": true
+            },
+            {
+              "autofocus": false,
+              "input": true,
+              "tableView": false,
+              "inputType": "radio",
+              "label": "Is your parent(s)/step-parent/sponsor/legal guardian a resident of B.C.?",
+              "key": "parentbcResident",
+              "values": [
+                {
+                  "value": "yes",
+                  "label": "Yes",
+                  "shortcut": ""
+                },
+                {
+                  "value": "no",
+                  "label": "No",
+                  "shortcut": ""
+                }
+              ],
+              "defaultValue": "",
+              "protected": false,
+              "fieldSet": false,
+              "persistent": true,
+              "hidden": false,
+              "clearOnHide": true,
+              "validate": {
+                "required": true,
+                "custom": "",
+                "customPrivate": false,
+                "customMessage": ""
+              },
+              "type": "radio",
+              "labelPosition": "top",
+              "optionsLabelPosition": "right",
+              "tags": [],
+              "conditional": {
+                "show": "true",
+                "when": "howWillYouBeAttendingTheProgram",
+                "eq": "Full Time"
```

Comment:
The condition to show this must not be full time. Rather it must be `dependantStatus === 'dependant'`